app/code/community/Bolt/Boltpay/Helper/Data.php

```diff
@@ -198,8 +197,6 @@ public function buildOnCloseCallback($closeCustom, $checkoutType)
                     ";
                 break;
             case Bolt_Boltpay_Block_Checkout_Boltpay::CHECKOUT_TYPE_PRODUCT_PAGE:
-                $quoteId = Mage::getSingleton('checkout/session')->getQuoteId();
```

Comment:
This is not needed anymore? @adentong @mccottry

Reply:
These are literally declared and then we just return. So definitely not used.

Reply:
sg then, but I'd like @mccottry to weigh in as well 👍

Reply:
@matt-thomason and @adentong 
The code above is unused and is wrong.  There is no meaningful session quote id in the context of product page checkout. What this code represents, (i.e. the close callback), is what happens if the server-side Bolt admin configuration has disabled pre-auth.  

In this instance, it is not just the two lines that should be removed, but actually the entire `case` for `Bolt_Boltpay_Block_Checkout_Boltpay::CHECKOUT_TYPE_PRODUCT_PAGE`.

Instead, it should rely on the `default` `case`.